src/main/java/homework_2/traffic_light/TrafficLight.java

```diff
@@ -0,0 +1,56 @@
+package homework_2.traffic_light;
+
+import java.util.NoSuchElementException;
+import java.util.Scanner;
+
+public class TrafficLight {
```

Comment:
If i insert 5 7, it doesn't give me an error

Reply:
Fixed. Also I added a test for this case